dev_env/README.md

```diff
@@ -0,0 +1,90 @@
+# Local Dev
+## Required Dependencies
+You need to install the following do deploy a development stack:
+
+    - python (3+)
+    - docker
+    - docker-compose
+    
+This also assumes you can run as root/administrator.  This is due to ports being opened & docker access.
+
+## How To
+1. Configure
+
+    To modify Govready-Q settings, you'll need to `generate an environment.json` to pass to the container.
```

Comment:
I ran this command with python3 since that's my python command. `python3 run.py init.py`. Other than that it worked as described.